bin/check.pl

```diff
@@ -244,23 +247,50 @@
     }
 }
 
-# Creating UDP socket to every Miniserver
-foreach my $ms (sort keys %miniservers) {
-    # Send value
-    my $sock = IO::Socket::INET->new(
-         Proto    => 'udp',
-         PeerPort => $udpport,
-         PeerAddr => $miniservers{$ms}{IPAddress},
-         Type        => SOCK_DGRAM
-    ) or lox_die "Could not create socket: $!";
-
-    for ($j=0;$j<$user_count;$j++) {
-        $sock->send("$users[$j]{NAME}:$users[$j]{ONLINE}") or lox_die "Send error: $!";
-        LOGINF "Sending Data '$users[$j]{NAME}:$users[$j]{ONLINE}' to $miniservers{$ms}{Name} IP: $miniservers{$ms}{IPAddress} Port:$udpport";
+# send Data
+
+if ($udp_enable) {
+    foreach my $ms (sort keys %miniservers) {
+        # Send value
+        my $sock = IO::Socket::INET->new(
+             Proto    => 'udp',
+             PeerPort => $udpport,
+             PeerAddr => $miniservers{$ms}{IPAddress},
+             Type        => SOCK_DGRAM
+        ) or lox_die "Could not create socket: $!";
+
+        for ($j=0;$j<$user_count;$j++) {
+            $sock->send("$users[$j]{NAME}:$users[$j]{ONLINE}") or lox_die "Send error: $!";
+            LOGOK "Sending Data '$users[$j]{NAME}:$users[$j]{ONLINE}' to $miniservers{$ms}{Name} IP: $miniservers{$ms}{IPAddress} Port:$udpport";
+        }
+        LOGEND "Operation finished sucessfully.";
+        $sock->close();
+    }
+} else {
+    ##MQTT publish
+
+    # Allow unencrypted connection with credentials
+    $ENV{MQTT_SIMPLE_ALLOW_INSECURE_LOGIN} = 1;
+
+    my $mqttcred = LoxBerry::IO::mqtt_connectiondetails();   
```

Comment:
Mhh das funktioniert nur ab Loxberry 2.0, damit das Plugin auch mit 1.x funktioniert, sollten wir entweder ne Moeglichkeit anbieten den MQTT server im Plugin einstellen zu können oder wir deaktivieren MQTT einfach komplett in dem Fall.

Reply:
Ja das stimmt.
Das Problem von meiner Seite ist, dass ich nur einen Loxberry auf 2.0.1.3 habe und den auch für die Steuerung benötige... ich kann leider nicht mehr downgraden um zu testen ob und wie das ginge... 
Der einfachere Weg ist eher die MQTT Funktionalität erst mit 2.0 freizuschalten!? Ich denke dass die mehreren MQTT Plugins sowieso einen umstieg auf 2.0 erforderlich machen!?

Reply:
Passt, dann mergen wir das so und ich fixe es auf meinem System, damit es ohne MQTT auch weiterhin funktioniert.

Reply:
Bzw. teste es nur, reicht ja eigentlich das zu dokumentieren...